tests/unit/test_magics.py

```diff
@@ -1970,3 +1970,19 @@ def test_bigquery_magic_create_dataset_fails():
         )
 
     assert close_transports.called
+
+
+@pytest.mark.usefixtures("ipython_interactive")
+def test_bigquery_magic_send_widget_job():
+
+    ip = IPython.get_ipython()
+    ip.extension_manager.load_extension("google.cloud.bigquery")
+
+    run_query_patch = mock.patch(
+        "google.cloud.bigquery.magics.magics._run_query", autospec=True
+    )
+
+    with run_query_patch as query_mock:
+        cell_magic_args = "--send_widget_job"
+        ip.run_cell_magic("bigquery", cell_magic_args, "SELECT 1")
```

Comment:
It would be nice if we could test a query that takes longer than 10 sec to complete and verify that the widget output shows the 'job still running' and 'job is done' status messages if possible. At the least, the status messages to indicate that the widget really kicked in and ran its code.

Reply:
Added a longer-running query to start!